astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -40,7 +40,8 @@ def cirs_to_altaz(cirs_coo, altaz_frame):
     xp, yp = get_polar_motion(obstime)
 
     #first set up the astrometry context for CIRS<->AltAz
-    astrom = erfa.apio13(obstime.jd1, obstime.jd2,
+    obstime_utc = obstime if obstime.scale == 'utc' else obstime.utc
```

Comment:
@taldcroft - I have on and off wrote sequences like this as well, as it helps speed up the case where the scale is already correct (avoiding the creation of a new instance). But it is also a bit ugly... Often, like here, it is in preparation to get both jd, so maybe a function to get those in a certain scale? (with default, just returning internal `jd1`, `jd2` if no scale given). But perhaps this is overoptimization...